UniConnectApp/app/src/main/java/com/example/uniconnect/MainActivity.kt

```diff
@@ -2,17 +2,19 @@ package com.example.uniconnect
 
 import android.os.Bundle
 import android.util.Log
+import android.widget.Toast
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
-import androidx.compose.foundation.layout.Column
-import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
```

Comment:
I'd keep this as it were.  Better to import each class separately, than to use wildcards.  Wildcards give an opportunity for class name collision, if a class name is duplicated across two packages that are imported with wildcards.  Importing each class is also better for documentation.  And, it does not cost anything performance-wise, as it all gets cleaned up by the compiler.